src/css/skins/seven.less

```diff
@@ -48,4 +48,8 @@
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
   }
+
+  &.jw-ie .jw-text-alt {
+    margin: 0.5em 0;
```

Comment:
The seven skin gets this from the control bar: https://github.com/jwplayer/jwplayer/blob/master/src/css/imports/controlbar.less#L40.

The goal is to implement a comprehensive solution for centering text in IE11 for all the other skins. Removing the margin from `ads.less` still doesn't get us there.